person.rb

```diff
@@ -2,7 +2,7 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
```

Comment:
- [ ] You need to define the `@rentals` variable within this initialize method, else we cannot add a rental to this Person class, like so: `@rentals = []`